app/controllers/subscriptions_controller.rb

```diff
@@ -0,0 +1,33 @@
+class SubscriptionsController < ActionController::Base
+  def create
+    subscription = Subscription.find_or_initialize_by(
+      subscriber: subscriber,
+      subscriber_list: subscribable,
+    )
+
+    if subscription.new_record?
+      subscription.save!
+      status = :created
+    else
+      status = :ok
+    end
+
+    render json: { id: subscription.id }, status: status
```

Comment:
I wonder should we use the `uuid` field here as we interact with other parts of the API (unsubscribe at the moment) using that. I've approved with this as is as I think it's fine but curious if anybody else has any opinions.

Reply:
I don't think it matters too much at this point - I'm usually in favour of not leaking incrementing numbers if we don't have to, but that's probably a false concern really since we can't actually use these against any of our endpoints to mutate data anyway (I think). 

I wonder if the returned Id is even needed? For unsubscribe, we just returned a status code with no body didn't we?

Reply:
I figured the ID might be useful for debugging if we have to manually trigger that endpoint for whatever reason.

Reply:
yep, probably will come in handy at some point :)